src/behaviors/entity-store.js

```diff
@@ -17,10 +18,11 @@ export class EntityStoreBehavior extends Behavior {
   items = [];
```

Comment:
"items" property became unreachable with the intoduction of a getter and a setter. Can we remove it from here?